tests/projection_test.py

```diff
@@ -16,11 +16,11 @@
 
 import jax
 import jax.numpy as jnp
-
+from jaxopt import OSQP
 from jaxopt import projection
 from jaxopt._src import test_util
 from jaxopt._src.lbfgs import LBFGS
-from jaxopt import OSQP
+from jaxopt._src.projection import projection_simplex_k_sparse  # temporary (until projection_simplex_k_sparse becomes public in JAXopt)  ## pylint: disable=line-too-long
```

Comment:
You can add `projection_sparse_simplex` here: https://github.com/google/jaxopt/blob/399224bfa50dff2e47c94275ca2a947f21e2f0ab/jaxopt/projection.py